commerce-address-web/src/main/java/com/liferay/commerce/address/web/internal/portlet/action/EditCommerceCountryMVCActionCommand.java

```diff
@@ -237,6 +239,12 @@ protected CommerceCountry updateCommerceCountry(ActionRequest actionRequest)
 		CommerceCountry commerceCountry = null;
 
 		if (commerceCountryId <= 0) {
+			if (_commerceCountryService.fetchCommerceCountry(
+					serviceContext.getCompanyId(), twoLettersISOCode) != null) {
+
+				throw new CommerceCountryAlreadyExistsException();
+			}
```

Comment:
Shouldn't this validation be inside `commerceCountryLocalService.addCommerceCountry`? There's a unique index, so I think we'll always want to run this validation.